pkg/minishift/util/fetch_iso_version.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+type VersionSpec struct {
+	Assets []struct {
+		Name               string `json:"name"`
+		BrowserDownloadURL string `json:"browser_download_url"`
+	} `json:"assets"`
+}
+
+func GetLatestISOVersion(isoName string) string {
```

Comment:
That's too simplistic really. The idea is not just to get the latest. We should also inspect the version and treat it as semantic version. Then we should determine whether this version is compatible with current Minishift.

What if we release a new non compatible change to the ISO? A change which required changes in Minishift as well? Users which have a older version of Minishift should then not automatically get this new version. 

The latest version of the ISO which is compatible in semantic versioning terms should be downloaded.